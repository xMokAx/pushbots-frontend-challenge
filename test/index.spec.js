import { mount } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import HomePage from '../pages/index.vue'

Vue.use(Vuetify)

describe('Home Page', () => {
  it('Should match snapshot', () => {
    const wrapper = mount(HomePage, {
      stubs: ['nuxt-link']
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
