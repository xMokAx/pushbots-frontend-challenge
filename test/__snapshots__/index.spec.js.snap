// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Should match snapshot 1`] = `
"<main class=\\"d-flex flex-grow-1 flex-column-reverse flex-md-row justify-center\\"><svg fill=\\"#fff\\" width=\\"140\\" height=\\"28\\" viewBox=\\"0 0 153 31\\" class=\\"logo\\"><g fill-rule=\\"evenodd\\"><path d=\\"M5.913 31h5.652v-3.296H5.913zM0 19.954h3.217v-7.75H0zM13.913 31h5.652v-3.296h-5.652zM.522 23.829c0 1.5 1.207 2.717 2.695 2.717v-5.434A2.706 2.706 0 00.522 23.83M22.783 19.954H26v-7.75h-3.217zM22.783 21.112v5.434a2.707 2.707 0 002.695-2.717c0-1.5-1.207-2.717-2.695-2.717M9.477 7.926c-1.211 0-2.193-1.007-2.193-2.248 0-1.242.982-2.25 2.193-2.25s2.192 1.008 2.192 2.25c0 1.241-.981 2.248-2.192 2.248zm6.959-4.497c1.211 0 2.193 1.007 2.193 2.249 0 1.241-.982 2.248-2.193 2.248-1.21 0-2.193-1.007-2.193-2.248 0-1.242.982-2.25 2.193-2.25zm-6.3 7.706h5.64v-1.38h4.92V1.6H15.9A3.559 3.559 0 0012.942 0a3.56 3.56 0 00-2.96 1.6H5.218v8.155h4.92v1.38z\\"></path> <path d=\\"M9.479 4.543c-.624 0-1.131.518-1.131 1.158 0 .64.507 1.158 1.13 1.158.624 0 1.13-.518 1.13-1.158 0-.64-.506-1.158-1.13-1.158M16.522 6.86c.624 0 1.13-.52 1.13-1.159 0-.64-.506-1.158-1.13-1.158-.624 0-1.13.518-1.13 1.158 0 .64.506 1.158 1.13 1.158M4.26 13.006v12.738l6.262-6.369zM20.957 12.204H5.043L13 20.31z\\"></path> <path d=\\"M13 21.896l-.004-.01-.002.004-1.71-1.758-6.24 6.414h15.913l-6.241-6.414zM21.74 25.744V13.006l-6.262 6.369z\\"></path> <g><path d=\\"M35.536 21.069h2.73c.871 0 1.264-.028 1.442-.2.28-.247.412-.603.412-1.873 0-1.23-.145-1.591-.414-1.825-.174-.145-.413-.2-1.44-.2h-2.73v4.098zm-4.482 9.42c-.582 0-1.054-.46-1.054-1.023V14.022c0-.564.472-1.022 1.054-1.022h9.786c1.666 0 2.73.246 3.346.774.857.733 1.648 1.876 1.648 5.247 0 3.372-.791 4.514-1.648 5.246-.616.528-1.68.775-3.346.775h-5.304v4.424c0 .564-.472 1.022-1.054 1.022h-3.428zM51.37 31c-1.26 0-2.096-.247-2.628-.778-.62-.615-.808-1.447-.808-3.563v-8.708a1.03 1.03 0 011.033-1.027h3.062c.57 0 1.035.46 1.035 1.027v8.031c0 .505.07.773.252.955.184.181.424.25.885.25 1.397 0 2.25-.327 2.25-.327v-8.909a1.03 1.03 0 011.033-1.027h3.063c.57 0 1.034.46 1.034 1.027V29.95a1.03 1.03 0 01-1.034 1.026h-3.063a1.03 1.03 0 01-1.033-1.026v-1.03C54.66 30.67 54.05 31 52.609 31H51.37zM67.769 31c-1.493 0-2.19-.185-2.661-.705-.485-.533-.714-1.296-.813-2.72a1.03 1.03 0 011.034-1.1h2.428c.46 0 .865.3.995.74.017.059.04.106.079.164.108.178.49.178.633.178h2.533c.367 0 .57-.039.66-.127.09-.09.1-.293.1-.703 0-.38-.037-.54-.152-.654-.114-.113-.391-.182-1.014-.251l-4.43-.552c-1.007-.117-1.579-.336-2.026-.78-.54-.535-.709-1.041-.709-3.167 0-2.226.594-3.278 1.09-3.771.447-.445 1.04-.628 2.05-.628h6.102c1.7 0 2.408.365 2.701.672.46.454.66 1.156.751 2.663a1.028 1.028 0 01-1.034 1.09h-2.457c-.43 0-.814-.264-.968-.661a.702.702 0 00-.075-.134c-.114-.186-.47-.186-.639-.186h-2.153c-.412 0-.553.02-.683.15-.089.087-.127.192-.127.579 0 .357.032.54.227.68.138.079.289.126.887.2l4.658.552c.969.115 1.56.34 1.976.756.492.487.683 1.39.683 3.215 0 1.877-.364 3.152-1.112 3.896-.434.428-1.026.604-2.053.604H67.77zM89.154 30.488a1.03 1.03 0 01-1.035-1.022v-7.922c0-.503-.072-.77-.252-.95-.185-.182-.424-.25-.886-.25h-.429a4.5 4.5 0 00-1.16.16 1.59 1.59 0 01-.122.022c-.038.016-.39.121-.54.168l.002 8.772a1.03 1.03 0 01-1.035 1.022h-3.061a1.028 1.028 0 01-1.034-1.022V14.022c0-.564.463-1.022 1.034-1.022h3.061c.572 0 1.035.458 1.035 1.022v4.549c1.76-1.765 2.372-2.099 3.842-2.099h1.239c1.258 0 2.093.246 2.629.774.619.613.808 1.442.808 3.55v8.67c0 .564-.463 1.022-1.034 1.022h-3.062zM100.95 19.87h3.648c.491 0 .692-.015.859-.175.228-.25.365-.727.365-1.374 0-.612-.114-1.033-.34-1.25-.149-.143-.397-.2-.884-.2h-3.648v2.999zm0 6.745h3.725c.512 0 .767-.06.939-.225.207-.197.365-.426.365-1.4 0-.964-.12-1.262-.34-1.473-.187-.18-.457-.25-.964-.25h-3.725v3.348zm-4.534 3.873c-.589 0-1.067-.458-1.067-1.022V14.022c0-.564.478-1.022 1.067-1.022h10.343c1.651 0 2.752.317 3.465.998.857.822 1.33 2.029 1.33 3.398 0 2.443-.915 3.645-3.153 4.148 2.04.503 3.307 1.368 3.307 4.272 0 1.548-.45 2.734-1.38 3.623-.735.705-1.894 1.05-3.543 1.05H96.416zM119.279 27.028c.154.201.359.226.897.226h1.436c.538 0 .744-.05.897-.226.154-.175.256-.603.256-3.066s-.102-2.89-.256-3.067c-.153-.176-.359-.225-.897-.225h-1.436c-.538 0-.743.024-.897.225-.154.177-.257.604-.257 3.067s.103 2.89.257 3.066m-3.999 3.067c-.949-.955-1.385-2.388-1.385-6.133 0-3.744.436-5.177 1.385-6.133.64-.678 1.36-.905 2.871-.905h5.487c1.513 0 2.204.227 2.871.905.949.956 1.384 2.389 1.384 6.133 0 3.77-.435 5.178-1.384 6.133-.667.679-1.358.905-2.871.905h-5.487c-1.512 0-2.205-.226-2.87-.905M143.354 31c-1.493 0-2.19-.185-2.66-.705-.486-.533-.715-1.296-.814-2.72a1.037 1.037 0 011.035-1.1h2.428c.46 0 .864.3.994.74.018.059.04.106.08.164.107.178.489.178.633.178h2.531c.368 0 .572-.039.66-.127.09-.09.1-.293.1-.703 0-.38-.037-.54-.153-.654-.114-.113-.39-.182-1.013-.251l-4.43-.552c-1.005-.117-1.578-.336-2.026-.78-.538-.535-.706-1.041-.706-3.167 0-2.226.591-3.278 1.089-3.771.445-.445 1.04-.628 2.049-.628h6.1c1.7 0 2.408.365 2.702.672.459.454.661 1.156.751 2.663a1.036 1.036 0 01-1.034 1.09h-2.458c-.428 0-.813-.264-.966-.661a.662.662 0 00-.075-.134c-.115-.186-.469-.186-.641-.186h-2.15c-.414 0-.553.02-.685.15-.087.087-.126.192-.126.579 0 .357.032.54.227.68.14.079.288.126.886.2l4.659.552c.967.115 1.558.34 1.976.756.49.487.683 1.39.683 3.215 0 1.877-.366 3.152-1.115 3.896-.432.428-1.025.604-2.05.604h-6.481zM136.617 20.368c.592 0 1.074-.467 1.074-1.044v-1.711c0-.576-.482-1.044-1.074-1.044h-2.44v-2.526c0-.576-.481-1.043-1.073-1.043h-3.176c-.59 0-1.073.467-1.073 1.043V27.2c0 1.83.282 2.469.734 2.907.428.415 1.159.892 3.043.892 1.587 0 2.996-.199 3.775-.382l.426-.096c.487-.11.831-.53.831-1.016v-1.44c0-.575-.481-1.043-1.073-1.043h-1.156c-.56 0-.893-.028-1.047-.18-.156-.15-.211-.403-.211-.967v-5.508h2.44z\\"></path></g></g></svg> <section class=\\"primary flex-grow-1 flex-shrink-0 width-md-half\\"><div class=\\"white--text d-flex justify-center align-center v-card v-sheet theme--light elevation-0 primary\\" style=\\"height: 100%; min-height: 480px;\\"><h2 class=\\"font-weight-regular title\\">Testimonials</h2></div></section> <section class=\\"white flex-grow-1 flex-shrink-0 width-md-half d-flex flex-column justify-center align-center form-wrapper px-7 py-md-4 px-md-6 elevation-4\\"><div class=\\"v-card v-sheet theme--light elevation-0\\" style=\\"max-width: 500px; width: 100%;\\"><h2 class=\\"font-weight-regular headline mb-4\\">Log into your account</h2> <form novalidate=\\"novalidate\\" class=\\"v-form d-flex flex-column\\"><div class=\\"v-input theme--light v-text-field\\"><div class=\\"v-input__control\\"><div class=\\"v-input__slot\\"><div class=\\"v-text-field__slot\\"><label for=\\"input-7\\" class=\\"v-label theme--light\\" style=\\"right: 0px; position: absolute;\\">E-mail</label><input name=\\"email\\" required=\\"required\\" id=\\"input-7\\" type=\\"text\\"></div></div><div class=\\"v-text-field__details\\"><div class=\\"v-messages theme--light\\"><span name=\\"message-transition\\" tag=\\"div\\" class=\\"v-messages__wrapper\\"></span></div></div></div></div> <div class=\\"v-input theme--light v-text-field\\"><div class=\\"v-input__control\\"><div class=\\"v-input__slot\\"><div class=\\"v-text-field__slot\\"><label for=\\"input-10\\" class=\\"v-label theme--light\\" style=\\"right: 0px; position: absolute;\\">Password</label><input name=\\"password\\" required=\\"required\\" id=\\"input-10\\" type=\\"password\\"></div></div><div class=\\"v-text-field__details\\"><div class=\\"v-messages theme--light\\"><span name=\\"message-transition\\" tag=\\"div\\" class=\\"v-messages__wrapper\\"></span></div></div></div></div> <nuxt-link-stub to=\\"/register\\" class=\\"body-2 font-weight-bold\\">Don't have an account?</nuxt-link-stub> <nuxt-link-stub to=\\"/reset-pw\\" class=\\"body-2 font-weight-bold\\">Forgot your password?</nuxt-link-stub> <button type=\\"submit\\" disabled=\\"disabled\\" class=\\"my-4 v-btn v-btn--block v-btn--contained v-btn--disabled theme--light v-size--default\\"><span class=\\"v-btn__content\\">
    LOGIN
  </span></button></form> <div class=\\"d-flex\\"><button type=\\"button\\" class=\\"google-btn mr-2 white--text flex-grow-1 v-btn v-btn--contained v-btn--tile theme--light v-size--default\\"><span class=\\"v-btn__content\\">
          Google
        </span></button> <button type=\\"button\\" class=\\"github-btn white--text flex-grow-1 v-btn v-btn--contained v-btn--tile theme--light v-size--default\\"><span class=\\"v-btn__content\\">
          Github
        </span></button></div></div></section> <div class=\\"v-overlay theme--dark\\" style=\\"z-index: 5;\\"><div class=\\"v-overlay__scrim\\" style=\\"opacity: 0; background-color: rgba(255, 255, 255, 0.7);\\"></div></div></main>"
`;
